proposals/csharp-9.0/records.md

```diff
@@ -13,9 +13,11 @@ record_declaration
     ;
 
 record_base
-    : ':' class_type argument_list?
-    | ':' interface_type_list
-    | ':' class_type argument_list? ',' interface_type_list
+    : ':' class_type record_base_arguments? (',' interface_type_list)?
+    ;
+
+record_base_arguments
+    : '(' argument_list? ')'
```

Comment:
This can be further simplified because `argument_list` does include the `(` and `)`:

```suggestion
    : ':' class_type argument_list? (',' interface_type_list)?
    ;
```